web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -104,6 +101,8 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo }: TabbedDataGridProp
     mouseY: number
   } | null>(null)
 
+  const storedDraftForecast = new MorecastDraftForecast(localStorage)
+
```

Comment:
@dgboss Curious how you would write this into a local state var when the useState setter isn't needed

Reply:
Would defining it this way but outside the component be a reasonable solution? It shouldn't instantiate a new object on every rerender then I don't think?

Reply:
Yes, this is perfectly reasonable and some would argue a better approach I'm sure. The alternative is to do `const [storedDraftForecast] = useState(new MorecastDraftForecast(localStorage))`